src/community/wps-longitudinal-profile/src/main/java/org/geoserver/wps/longitudinal/LongitudinalProfileProcess.java

```diff
@@ -96,11 +127,14 @@ public void destroy() throws Exception {
             meta = {"mimeTypes=application/json"},
             type = LongitudinalProfileProcessResult.class)
     public LongitudinalProfileProcessResult execute(
-            @DescribeParameter(name = "layerName", description = "Input raster name", min = 1) String layerName,
+            @DescribeParameter(name = "layerName", description = "Input raster name", min = 0) String layerName,
+            @DescribeParameter(name = "coverage", description = "Input coverage", min = 0) GridCoverage2D coverage,
```

Comment:
coverage input parameter is introduced but no documentation addition on:
https://docs.geoserver.org/main/en/user/community/wps-longitudinal-profile/index.html